app/src/main/res/values/strings.xml

```diff
@@ -819,6 +819,14 @@
     <string name="weather_kind_thunder">Thunder</string>
     <string name="weather_kind_thunderstorm">Thunderstorm</string>
     <string name="weather_kind_wind">Wind</string>
+    <string name="weather_kind_mist">Mist</string>
+    <string name="weather_kind_mainly_cloudy">Mainly cloudy</string>
+    <string name="weather_kind_dry">Dry</string>
+    <string name="weather_kind_humid">Humid</string>
+    <string name="weather_kind_hot">Hot</string>
+    <string name="weather_kind_warm">Warm</string>
+    <string name="weather_kind_cool">Cool</string>
+    <string name="weather_kind_cold">Cold</string>
```

Comment:
These weather kind don't exist in Breezy Weather. If they are HKO-specific, move them as `hko_` and `translatable="false"`.

Reply:
Sure. I'll move these. I have a question and a comment though:

1. What's the difference between the `common_weather_text_` labels and the `weather_kind_` labels?
2. A few of these weather types will become shared with other sources. If I remember correctly, MGM (Türkiye) will share "mist", "hot" and "cold" with HKO. While "smoke" (currently coded for Meteo AM) will be used by MGM as well.